gradio/blocks.py

```diff
@@ -1845,7 +1845,7 @@ def reverse(text):
                     analytics.error_analytics("Not able to set up tunnel")
                 self.share_url = None
                 self.share = False
-                print(strings.en["COULD_NOT_GET_SHARE_LINK"])
+                print(strings.en["COULD_NOT_GET_SHARE_LINK"].format(BINARY_FILENAME))
```

Comment:
I think it would be better if we displayed BINARY_PATH so that users don't have to search for the full path themselves. They may not know where the site-packages directory is on their machine.